app/src/main/res/values-es/strings.xml

```diff
@@ -396,4 +396,55 @@
+    <string name="downloadConfirmationOpenFileText">Abrir</string>
+    <string name="downloadConfirmationUnableToOpenFileText">No se ha podido abrir el archivo</string>
+    <string name="downloadConfirmationUnableToDeleteFileText">No ha podido eliminar el archivo antiguo</string>
+
+    <!-- Change Icon Activity -->
+    <string name="changeIconActivityTitle">Icono de la aplicación</string>
+    <string name="changeIconCtaLabel">Cambiar icono de la aplicación</string>
+    <string name="changeIconCtaAccept">Usar</string>
+    <string name="changeIconCtaCancel">Cancelar</string>
+    <string name="changeIconDialogTitle">¿Usar nuevo ícono?</string>
+    <string name="changeIconDialogMessage">La aplicación puede cerrarse para actualizar los cambios. Vuelve después de que hayas admirado tu nuevo y atractivo icono.</string>
+
+    <!-- New dashboard / menu buttons -->
+    <string name="manageWhitelist">Administrar lista blanca</string>
+    <string name="reportBrokenSite">Reportar sitio web roto</string>
+
+    <!-- Dialogs -->
+    <string name="dialogAddTitle">Añadir</string>
+    <string name="dialogEditTitle">Editar</string>
+
+    <!-- Whitelist -->
+    <string name="whitelisetActivityTitle">Lista blanca</string>
+    <string name="whitelistAdd">Agregar a la lista blanca</string>
+    <string name="whitelistRemove">Eliminar de la lista blanca</string>
+    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio web de la lista blanca %s</string>
+    <string name="whitelistEntryDeleteConfirmMessage">¿Estás seguro de que desea eliminar &lt;b&gt;%s&lt;/b&gt; de la lista blanca?</string>
+    <string name="whitelistExplanation">Estos sitios web incluidos en la lista blanca no se actualizarán por Protección de Privacidad</string>
+    <string name="whitelistNoEntries">Aún no hay sitios web en la lista blanca</string>
+    <string name="whitelistDomainHint">www.ejemplo.com</string>
+    <string name="whitelistFormatError">No se ha podido guardar, ingresa un dominio como <b>example.com</b>
+  </string>
+
+    <!-- Settings -->
+    <string name="settingsPrivacyProtectionWhitelist">Lista blanca de Protección de Privacidad</string>
+
+    <!-- Fireproof websites -->
+    <string name="fireproofWebsitesActivityTitle">Sitios web a prueba de fuego</string>
+    <string name="settingsFireproofWebsites">Sitios web a prueba de fuego</string>
+    <string name="fireproofWebsiteMenuTitle">Sitio web a prueba de fuego</string>
+    <string name="fireproofWebsiteSnackbarConfirmation">¡&lt;b&gt;%s&lt;/b&gt; ahora es a prueba de fuego! Visita Ajustes para obtener más información.</string>
+    <string name="fireproofWebsiteSnackbarAction">Deshacer</string>
+    <string name="fireproofWebsiteDeleteConfirmMessage">¿Está seguro de que desea eliminar &lt;b&gt;%s&lt;/b&gt;?</string>
+    <string name="fireproofWebsiteEmptyListHint">No hay sitios web a prueba de fuego todavía</string>
```

Comment:
`Todavía no hay sitios web a prueba de fuego`? Using todavía at the end seems a direct translation from `yet`.